components/eam/bld/build-namelist

```diff
@@ -922,10 +927,9 @@ if ($co2_cycle) {
 	# (user specification has higher precedence than the true value set above)
 	if ($nl->get_value('co2_readflux_fuel') =~ /$TRUE/io) {
 	    add_default($nl, 'co2flux_fuel_file', 'sim_year'=>$sim_year);
-	    add_default($nl, 'co2flux_fuel_vrt_emis_file', 'sim_year'=>$sim_year);
```

Comment:
Do the changes to the handling of aircraft emissions belong in this PR?

Reply:
Same as before; this is not needed.